.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "editor.formatOnSave": false,
+  "editor.formatOnSave": true,
```

Comment:
my personal thought is that we should exclude editor settings from the repo. Not everybody uses vscode, so why should they have settings for it.

Reply:
Yeah but most use it and this one is pretty important configuration. No one is going to use vscoee without this config in this project.

Reply:
But I could delete this and add some recommended stuff to README for vscode

Reply:
Just checked that basically all city-of-helsinki projects have this directory included. It has all the basic setting to get vscode work nicely with our setup so I wouldn't remove it. All vscode users would need to add those settings anyway